src/utils/index.js

```diff
@@ -0,0 +1,47 @@
+import classNames from 'classnames';
+import { createContext } from 'react';
```

Comment:
let's not have `utils` folder in this app, you are already mixing unrelated logic here.  `AuthContext` should be a part of store layer and the validation logic should be used by smart form-like components that will render these inputs. the styling logic seems to be ui-only so please integrate it there